webapi/src/main/scala/org/knora/webapi/responders/v2/StandoffResponderV2.scala

```diff
@@ -1225,49 +1156,49 @@ class StandoffResponderV2(
    * @param resourceIri          the IRI of the resource containing the value.
    * @param valueIri             the IRI of the value.
    * @param offset               the start index of the first standoff tag to be returned.
-   *
    * @param requestingUser       the user making the request.
    */
   case class GetStandoffTask(
     resourceIri: IRI,
     valueIri: IRI,
     offset: Int,
     requestingUser: UserADM
-  ) extends Task[StandoffTaskUnderlyingResult] {
-    override def runTask(
-      previousResult: Option[TaskResult[StandoffTaskUnderlyingResult]]
-    )(implicit timeout: Timeout, executionContext: ExecutionContext): Future[TaskResult[StandoffTaskUnderlyingResult]] =
+  ) extends NextExecutionStep[StandoffTaskUnderlyingResult] {
+    override def run(
+      previousResult: Option[ResultAndNext[StandoffTaskUnderlyingResult]]
+    ): zio.Task[ResultAndNext[StandoffTaskUnderlyingResult]] =
```

Comment:
Since you have replaced the `org.knora.webapi.util.Task` with the `NextExecutionStep` please make sure you import  `zio.Task`.
```suggestion
    ): Task[ResultAndNext[StandoffTaskUnderlyingResult]] =
```